webapp/app/containers/MigrationHistory/Create/actions.js

```diff
@@ -47,7 +47,7 @@ export function getAllImages() {
           browserHistory.push('/login');
         }
         if (err || res.body.error) {
-          dispatch(getAllImagesResult({ images: [] }));
+          dispatch(getAllImagesResult([]));
```

Comment:
Yes, because in the case there is no error, getAllImagesResult() return an array, but here when there is an error it return an object.